proto/vtctldata.proto

```diff
@@ -209,16 +209,25 @@ message Shard {
   topodata.Shard shard = 3;
 }
 
+enum ShardedAutoIncrementHandling {
+  LEAVE = 0;
```

Comment:
WDYT of calling this `NONE` instead of `LEAVE`? Meaning we do nothing.

Reply:
I'm fine either way. If you prefer that then I'm find changing it.

Reply:
Let's change it then.

Reply:
Now I remember that this is what I had started with, but you cannot:
```
vtctldata.proto:213:3: "NONE" is already defined in "vtctldata".
```

See: https://github.com/protocolbuffers/protobuf/issues/5425

So we can leave it at LEAVE, or change it to e.g. NO_HANDLING etc.